examples/resources/astro_api_token/resource.tf

```diff
@@ -56,4 +56,31 @@ resource "astro_api_token" "example_workspace_token_with_deployment_role" {
       "entity_id" : "clyn6kxud003x01mtxmccegnh",
       "entity_type" : "DEPLOYMENT"
   }]
-}
\ No newline at end of file
+}
+
+resource "astro_api_token" "example_deployment_token" {
+  name        = "deployment api token"
+  description = "deployment api token description"
+  type        = "DEPLOYMENT"
+  roles = [{
+    "role" : "DEPLOYMENT_ADMIN",
+    "entity_id" : "clyn6kxud003x01mtxmccegnh",
+    "entity_type" : "DEPLOYMENT"
+  }]
+}
+
+resource "astro_api_token" "example_deployment_token_with_custom_role" {
+  name        = "deployment api token with custom role"
+  description = "deployment api token description"
+  type        = "DEPLOYMENT"
+  roles = [{
+    "role" : "DEPLOYMENT_ADMIN",
+    "entity_id" : "clyn6kxud003x01mtxmccegnh",
+    "entity_type" : "DEPLOYMENT"
+    },
```

Comment:
can remove this since it can only have one role

Reply:
won't it still need `entity_id` and `entity_type`? that's also how the type is setup so those are required

Reply:
ya its from the custom  role

Reply:
like just have the one custom role that has all 3 fields but roles array should be length 1